server/chat-commands/info.ts

```diff
@@ -912,40 +882,92 @@ export const commands: Chat.ChatCommands = {
 			targets.pop();
 		}
 
+		const originalSearch = target;
+		let imperfectMatch = false;
+		let isMatch = false;
 		let species: {types: string[], [k: string]: any} = dex.species.get(targets[0]);
-		const type1 = dex.types.get(targets[0]);
-		const type2 = dex.types.get(targets[1]);
-		const type3 = dex.types.get(targets[2]);
+		let type1: any = dex.types.get(targets[0]);
```

Comment:
```suggestion
		let type1 = dex.types.get(targets[0]);
```
This shouldn't need to be `any`. Let TypeScript infer it - will allow for better error catching.

Reply:
Done. I also removed the explicit typing from the species declaration as well. Let me know if you'd prefer it the other way. Thanks for the feedback!

Reply:
Fixed the linting issues by looping through the species types instead of adding them as an array. Let me know if there's a better way to accomplish that or if there's anything else I can do. Thanks!